utils/logutils.go

```diff
@@ -0,0 +1,12 @@
+package utils
+
+import (
+	"os"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+func LogExit(msg string, fmts ...interface{}) {
+	log.Fatalf(msg, fmts...)
```

Comment:
Thanks Simon! @mapuri I’ll start fixing this up tonight, spare yourself some effort for now, we can discuss it tomorrow if necessary.

-Erik

> On May 26, 2015, at 5:10 PM, Simon Eskildsen <notifications@github.com> wrote:
> 
> In utils/logutils.go <https://github.com/contiv/netplugin/pull/79#discussion_r31092717>:
> 
> > @@ -0,0 +1,12 @@
> > +package utils
> > +
> > +import (
> > +	"os"
> > +
> > +	log "github.com/Sirupsen/logrus"
> > +)
> > +
> > +func LogExit(msg string, fmts ...interface{}) {
> > +	log.Fatalf(msg, fmts...)
> Both merged!
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/contiv/netplugin/pull/79/files#r31092717>.
>